src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,12 @@
+package racingcar.domain
+
+data class Car(
+  val name: String, private var _distance: Int = 0
+) {
```

Comment:
tmi 일수도 있지만 전 이런식으로 모든 매개변수를 줄바꿈하는 편입니다!
```suggestion
data class Car(
  val name: String,
  private var _distance: Int = 0
) {
```

Reply:
코틀린 코딩 컨벤션에 대한 내용이군요!
적용하신다면 추가로 trailing comma도 적용해보시면 좋을 것 같습니다.
https://kotlinlang.org/docs/coding-conventions.html#trailing-commas